source/docs/application-notes/manual-pid-tuning.rst

```diff
@@ -0,0 +1,299 @@
+   9. In general, the system wants as high a kP gain as possible to decrease the time taken to get to the setpoint. This also requires a high kD gain to properly dampen the system. The limit of how high the kP/kD term can be is determined by the system latency, at which point the oscillation is impossible to avoid. The goal of repeating steps 7 and 8 is to find that limit.
+   10. Always verify the gains work for the expected setpoints of the system, it's possible the general solution may not work under the expected operating range of the system. If that's the case, re-tune for the expected operating range using the generic gains as a basis.
+
+.. raw:: html
+
+    <div class="viz-div" id="vertical_arm_container">
+      <div >
+         <div class="col" id="vertical_arm_plotVals"></div>
+         <div class="col" id="vertical_arm_plotAmps"></div>
+      </div>
+      <div class="flex-grid">
+         <div class="col" id="vertical_arm_viz"></div>
+         <div id="vertical_arm_ctrls"></div>
+      </div>
+      <script>
+         turret = new VerticalArmPIDF("vertical_arm", "both");
+      </script>
+    </div>
+
+.. dropdown:: Tuning Process Example
+
+   Following the guide, I start with all gains at 0 to dial in kG.
+
+   I start with a kG of 1, and notice that the arm's still falling, so I increase it to 2, 4, 8, and 16 before it stops moving. From there I reduce it to 12, then 10 and notice it fall again. I bring it up to 11 and see it still falls appreciably, so I leave it at 12 for the lower bound.
+
+   Going back up, I start at 16 again, then to 18, and 20 before it moves its way up. 19 also produces appreciable movement, so I leave it at 18. This means my kG is (12 + 18) / 2 = 15 amps.
+
+   From here, I set a setpoint of 0.1 and dial in kS to just before it starts moving. I increase it to 1, 2, and 4 when it starts moving. From here, I dial it down to 3 where it doesn't move, and back up to 3.5, 3.7 where it moves again. I check 3.6 and see it doesn't move, so I leave kS at 3.6 amps.
+
+   Now it's time for kP/kD tuning. I bring kP up to 1, 2, 4, 8, 16, and 32 before I get significant overshoot, where I dial kD in to 1, 2, 4, and 8 before that overshoot is gone. kP keeps increasing to 64 and 128, then kD goes up to 16 and 32 before it's back to kP. I go up to 256 and 512 where I notice a bit of oscillation, and I may be near the limit at this point. kD increases to 64 and I get oscillation on the way to the target, so I bring it down to 50 then 40 before I'm happy with it. There's still a little oscillation at the target, but it's minimal.
+
+   I check with other setpoints of -0.1, 0.4, 0.6 and confirm the movement looks good, and say the PID tuning is done.
+
+Profiled Tuning
+^^^^^^^^^^^^^^^
+Profiled tuning can be treated much the same way as normal tuning, but the introduction of a profile the system should follow means much of the response can be calculated in advance with feed-forwards.
```

Comment:
```suggestion
Profiled tuning follows a similar process as normal tuning, but with the introduction of a motion profile, this means much of the response can be calculated in advance with feed-forwards.
```

You don't have to use my recommendation, but the sentence is stiff. The latter half is particularly stiff.